api/app/src/main/kotlin/packit/clients/GithubUserClient.kt

```diff
@@ -22,13 +19,10 @@ class GithubUserClient(private val config: AppConfig, private val githubBuilder:
         ghUser = getGitHubUser()
     }
 
-    fun getUserPrincipal(): UserPrincipal
+    fun getGithubUser(): GHMyself
```

Comment:
the class that calls it is called GithbApiLoginService... thus It is a GH class and feel like its appropriate.. because its using the GithubClient class anyways alot